cardano-api/test/cardano-api-test.cabal

```diff
@@ -18,17 +18,12 @@ common project-config
                         OverloadedStrings
 
   ghc-options:          -Wall
+                        -Wcompat
                         -Wincomplete-record-updates
                         -Wincomplete-uni-patterns
-                        -Wredundant-constraints
                         -Wpartial-fields
-                        -Wcompat
-
-  if impl(ghc >= 8.10)
-    ghc-options:        -Wunused-packages
-
-  if impl(ghc < 8.10)
```

Comment:
Should we update the docs that we expect >= 8.10?

Reply:
Actually, yeah.

Reply:
I've updated one place I've found in the documentation.